Exercise/compute_lowest_terms.py

```diff
@@ -1,4 +1,28 @@
-## TODO: complete the function "lowest_terms" below
+# TODO: complete the function "lowest_terms" below
 
 def lowest_terms(x):
-	return ""
+	x = x.split('/')
+	l = []
+	fac = []
+	res = []
+	for i in x:
+		l.append(int(i))
+	print(l)
```

Comment:
printing to screen for your own debugging process should not be in the final release code